cmd/check.go

```diff
@@ -11,11 +11,18 @@ import (
 	"github.com/elastic/elastic-package/internal/cobraext"
 )
 
+const checkLongDescription = `Use this command to verify if the package is correct in terms of formatting, validation and building.
+
+It will execute the format, lint, and build commands all at once, in that order. The goal of this command is to perform exactly same actions as the CI job runner to verify package contents.
```

Comment:
I would remove the last sentence about CI. That is coupling `elastic-package` to the `integrations` repo CI too strongly IMO.

Reply:
Fixed